_sass/homepage/_text-cards.scss

```diff
@@ -0,0 +1,84 @@
+.textcards-section {
+  flex-direction: column;
+  align-items: center;
+
+  @media screen and (max-width: 768px) {
+    padding: 6rem 2.5rem 6rem 2.5rem;
+  }
+}
+
+.textcards-subtitle {
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.textcards-title {
+  font-size: 3rem;
+  font-weight: 700;
+  line-height: 4rem;
+}
+
+.textcards-description {
+  font-size: 1.25rem;
+}
+
+.textcards-card-section {
+  justify-content: center;
+  align-items: flex-start;
+  gap: 2rem;
+  width: 100%;
+  
+  @media screen and (max-width: 1023px) {
```

Comment:
nit: are we able to switch to the globally defined breakpoints introduced in #314?

Reply:
Have swapped to the values in https://github.com/isomerpages/isomerpages-template/pull/313/commits/706b3472e15e47673c22c6300af3e073f5050449! Was unable to use the variables, they broke for remote build :(